crates/cli/src/commands.rs

```diff
@@ -58,3 +61,21 @@ pub struct RunScriptArgs {
     /// A pre-defined package script.
     pub command: String,
 }
+
+#[derive(Parser, Debug)]
+pub struct ListCommandArgs {
+    /// Indicates what dependencies to omit while running
+    /// ls
+    #[arg(long = "omit", short = 'O')]
+    pub omit: String
```

Comment:
It seems like `pnpm` has different characteristics than npm on this. Can you follow pnpm? https://pnpm.io/cli/list